source/PlayState.hx

```diff
@@ -1492,12 +1490,23 @@ class PlayState extends MusicBeatState
 			startAndEnd();
 			return;
 		}
-
-		var video:MP4Handler = new MP4Handler();
-		video.playVideo(filepath);
-		video.finishCallback = function()
+		else
```

Comment:
Is this `else` even set to anything?

Reply:
it is, it actually sets up the videos themselves